package.json

```diff
@@ -18,6 +18,7 @@
 		"deploy-stand-alone": "ts-node --project stand-alone/tsconfig-stand-alone.json stand-alone/deploy/deploy.ts",
 		"watch": "yarn run create-dist && NODE_ENV=development rollup -c --watch",
 		"app": "cd test-jw && yarn && yarn start",
+		"dev": "yarn build && yarn app",
```

Comment:
I've added `start` script that do the same thing on my branch ;)

Reply:
Nice, we need to choose which one to keep then.

Reply:
I actually avoided watch since @nandy-andy said we have experienced issues with it.

Reply:
Watch works fine for me. I have never noticed any issues

Reply:
I think the old readme mentioned issues but let's experience them by ourselves and then think about a fix ;)